### PR TITLE
Add LS deps

### PR DESCRIPTION
**Pull request by jonstorer**

### CODE REVIEW

File: mac [hunk 1]
```diff
@@ -15,11 +15,7 @@ append_to_zshrc() {
   local text="$1" zshrc
   local skip_new_line="${2:-0}"
 
-  if [ -w "$HOME/.zshrc.local" ]; then
```
Comment (jonstorer) [inline mac]: we probably don't need to change this.

File: mac [hunk 2]
```diff
@@ -121,42 +138,19 @@ brew "hub"
...
-brew "postgres", restart_service: true
-brew "redis", restart_service: true
+# running these from docker
+# brew "postgres", restart_service: true
+# brew "redis", restart_service: true
+# brew "rabbitmq", restart_service: true
+# brew "mongodb", restart_service: true
 EOF
-
-fancy_echo "Configuring Ruby ..."
```
Comment (jonstorer) [inline mac]: we can probably leave this in. I believe it just setups up ruby's bundler to be more efficient

**Comment by oscarjgarza** [comment]
lgtm

**Comment by noheltcj** [comment]
SO MUCH CLEARER THAN BOXEN

**Comment by cryptojuice** [comment]
LGTM

File: mac [hunk 3]
```diff
@@ -121,17 +142,20 @@ brew "hub"
...
+brew "nodenv"
+brew "node-build"
 brew "rbenv"
 brew "ruby-build"
 
 # Databases
 brew "postgres", restart_service: true
 brew "redis", restart_service: true
+brew "rabbitmq", restart_service: true
+brew "mongodb", restart_service: true
```
Comment (jonstorer) [inline mac]: @cryptojuice this is true, Brewfile does not manage versions, so we'll need to use docker for this. Do you have a docker image we can use as a team for specific versions?

Comment (cryptojuice) [inline mac]: @jonstorer All of the above have official docker images for specific versions. Once docker for mac/windows is installed, the command is as simple as ```docker run postgres:x.x.x ```. I can put together a short how-to for the team.

Comment (jonstorer) [inline mac]: @cryptojuice we talked IRL and we're going to create a single docker image that hosts everything.

File: mac [hunk 4]
```diff
@@ -95,24 +95,45 @@ fi
...
+brew "gcc"
+brew "phantomjs"
+brew "htop"
+brew "watch"
 
-# Heroku
-brew "heroku-toolbelt"
-brew "parity"
+# Docker
+brew "docker"
```
Comment (jonstorer) [inline mac]: do we want to add docker up? can we run that over and over?

Comment (jeremycrussell) [inline mac]: rm boot2docker stuff

File: mac [hunk 5]
```diff
@@ -121,19 +143,18 @@ brew "hub"
...
+brew "node-build"
 brew "rbenv"
 brew "ruby-build"
-
-# Databases
-brew "postgres", restart_service: true
-brew "redis", restart_service: true
 EOF
 
+boot2docker init
```
Comment (jeremycrussell) [inline mac]: rm

**Pull request closed without merging**